externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/KyuubiSparkUtil.scala

```diff
@@ -113,4 +113,35 @@ object KyuubiSparkUtil extends Logging {
       SparkSQLEngine.kyuubiConf.get(configEntry)
     }
   }
+
+  private lazy val yarnAppAttemptIdFromEnv: Option[Int] =
+    try {
+      val containerId = System.getenv("CONTAINER_ID")
+      if (containerId != null) {
+        // like: container_e34_1698816066031_5638072_01_000001
+        val items = containerId.split("_")
+        if (items.length == 6) {
+          Some(items(4).toInt)
+        } else {
+          None
+        }
+      } else {
+        None
+      }
+    } catch {
+      case t: Throwable =>
+        warn("Failed to get yarn app attempt id from environment", t)
+        None
+    }
+
+  def maybeAttempt(conf: SparkConf): Boolean = {
+    val master = conf.getOption("spark.master").getOrElse("")
+    val deployMode = conf.getOption("spark.submit.deployMode").getOrElse("")
```

Comment:
```suggestion
    val master = conf.get("spark.master", "")
    val deployMode = conf.get("spark.submit.deployMode", "")
```
The SparkConf's getter supports default value.

Reply:
thanks, fixed